tests/bugzilla/Bugzilla1110.java

```diff
@@ -31,6 +31,12 @@
 import org.w3c.dom.traversal.NodeIterator;
 import org.xml.sax.InputSource;
 
+// jkesselm May 2023: CAN NOT REPRODUCE; prints what the user says they expected to see.
```

Comment:
nit: CANNOT

Reply:
Nit: Either is correct. Unless there's a W3C style guide which says otherwise, I would rather leave this unchanged.